packages/ignite-basic-generators/templates/examples/GridExample.js.ejs

```diff
@@ -13,8 +11,8 @@ import styles from './Styles/GridExampleStyle'
 
 class GridExample extends React.Component {
 
-  state: {
-    dataSource: Object
+  state = {
```

Comment:
These were just a an interface, you won't need to convert these.  For the state ones, I'd say lets delete them.